app/assets/stylesheets/application.scss

```diff
@@ -24,10 +24,205 @@
 }
 
 .sr-only {
-    position:absolute;
-    left:-10000px;
-    top:auto;
-    width:1px;
-    height:1px;
-    overflow:hidden;
+    position: absolute;
+    left: -10000px;
+    top: auto;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
+}
+
+.print-this,
+.save-this {
+    border-bottom: 1px solid #0b0c0c;
+    margin: 0 0 15px;
+    display: inline-block;
+
+    &__link,
+    &__link:visited {
+        color: #0b0c0c;
+        font-size: 16px;
+        text-decoration: none;
+    }
+}
+
+.save-this {
+    margin-left: 1em;
+}
+
+.print-this__link,
+.save-this__link {
+    color: #000;
+    font-size: 16px;
+    padding-bottom: 20px;
+    text-decoration: none;
+    font-family: 'GDS Transport', Arial, sans-serif;
+}
+
+@page {
+    size: auto;
+    margin: 0;
+}
+
+.print-document {
+    font-family: GDS Transport, arial, sans-serif;
+    border: 1px solid #6f777b;
+    position: relative;
+    box-shadow: 6px 6px 0 0 #dee0e2;
+    padding: 25px 50px 40px 50px;
+    margin-bottom: 20px;
+    width: 75%;
+    min-height: calc(296mm - 100px);
+    page-break-after: always;
+
+    .grid-row {
+        margin: 0 -15px;
+    }
+
+    .grid-row::after {
+        clear: both;
+        content: '';
+        display: block;
+    }
+
+    .print-header {
+        margin-bottom: 15px;
+
+        .govuk-grid-column-one-third {
+            border-left: #28a197 solid;
+            padding-left: 8px;
+        }
+
+        h2 {
+            font-size: 1.4em;
+            font-weight: normal;
+            margin: 0;
+            line-height: 1;
+
+            span {
+                display: block;
+            }
+        }
+
+        h3 {
+            text-align: right;
+            margin: 2em 0 0;
+            color: #0b0c0c;
+            font-weight: 600;
+        }
+    }
+
+    .print-banner {
+        .govuk-panel--confirmation {
+            border: 2px solid #28a197;
+            display: block;
+            margin-bottom: 1em;
+            padding: 5px 10px;
+            background-color: #e1f3f1;
+            print-color-adjust: exact;
+
+            .govuk-panel__body {
+                color: #0b0c0c;
+                margin-top: 0;
+            }
+
+            .govuk-panel__title {
+                color: #0b0c0c;
+            }
+        }
+    }
+
+    @media (max-width: 40.0624em) {
+        .govuk-grid-column-one-third {
+            padding: 0;
+        }
+    }
+
+    hr {
+        display: none;
+    }
+
+    .footer-page-no {
+        position: absolute;
+        bottom: 0px;
+        left: 35px;
+    }
+
+    .got-ni-box {
```

Comment:
can this be named better....ni stands for national insurance

Reply:
changed to .about-results